gdsc-web-backend/Models/Enums/TypeEnum.cs

```diff
@@ -0,0 +1,8 @@
+namespace gdsc_web_backend.Models.Enums
+{
+    public enum TypeEnum
```

Comment:
Please rename TypeEnum into MenuItemTypeEnum, since we can have multiple models that have types (e.g. settings types)